.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Here you require what you didn't include in your repo. This causes an error when running rspec in a different machine.